src/plugins/ajs-destination/loader.ts

```diff
@@ -76,12 +85,17 @@ export async function loadIntegration(
 
 export async function unloadIntegration(
   name: string,
-  version: string
+  version: string,
+  obfuscate?: boolean
 ): Promise<void> {
   const pathName = normalizeName(name)
-  return unloadScript(
-    `${path}/integrations/${pathName}/${version}/${pathName}.dynamic.js.gz`
-  )
+  const obfuscatedPathName = obfuscatePathName(name, obfuscate)
+
+  const fullPath = `${path}/integrations/${
+    obfuscate ? obfuscatedPathName : pathName
+  }/${version}/${obfuscate ? obfuscatedPathName : pathName}.dynamic.js.gz`
```

Comment:
Same comment here wrt replacing the ternary expressions with the nullish expressions.